src/tests/functional/plugin/gpu/remote_blob_tests/cldnn_remote_blob_tests.cpp

```diff
@@ -219,6 +219,95 @@ TEST_P(RemoteBlob_Test, smoke_canInputPluginRemoteBlob) {
     }
 }
 
+TEST_P(RemoteBlob_Test, NV12toGrayscale) {
+#if defined(ANDROID)
+    GTEST_SKIP();
+#endif
+    const int num_batch = 1;
+    const int num_channels = 1;
+    const int height = 8;
+    const int width = 8;
+    auto fn_ptr_remote = ngraph::builder::subgraph::makeConvertTranspose({num_batch, num_channels, height, width});
+
+    CNNNetwork net_remote(fn_ptr_remote);
+
+    net_remote.getInputsInfo().begin()->second->setLayout(Layout::NCHW);
+    net_remote.getInputsInfo().begin()->second->setPrecision(Precision::U8);
+    net_remote.getInputsInfo().begin()->second->getPreProcess().setColorFormat(ColorFormat::NV12);
+    auto fake_image_data_y = FuncTestUtils::createAndFillBlob(net_remote.getInputsInfo().begin()->second->getTensorDesc(), 50, 0, 1);
+    auto fake_image_data_uv = FuncTestUtils::createAndFillBlob(net_remote.getInputsInfo().begin()->second->getTensorDesc(), 256, 0, 1);
```

Comment:
it should have 2x smaller size in h/w dimensions

Reply:
Fixed, thanks